helper-cli/src/main/kotlin/commands/GenerateRuleViolationResolutionsCommand.kt

```diff
@@ -58,16 +58,18 @@ internal class GenerateRuleViolationResolutionsCommand : CliktCommand(
 
     private val severity by option(
         "--severity",
-        help = "Only consider violations of the given severity. Allowed values: ERROR|WARNING|HINT."
-    ).enum<Severity>()
+        help = "Only consider violations of the given severity specified as comma separated values. Allowed values: " +
+                "ERROR|WARNING|HINT."
+    ).convert { value -> value.split(",").map { Severity.valueOf(it) }.toSet() }
+        .default(setOf(*Severity.values()))
```

Comment:
You could use `mapTo(enumSetOf())` (or `mapTo(mutableSetOf())` if you don't want to use enum sets) and `EnumSet.allOf(Severity::class.java)` or `enumSetOf(*Severity.value())`.

Reply:
Kept as is because type inference issues would lead to a bit more bloated code.

Reply:
But you could simply use `.enum<Severity>().split(",")` like we do for [outputFormats](https://github.com/oss-review-toolkit/ort/blob/fb71693509e8a01429f814be342ddad5086a5801/cli/src/main/kotlin/commands/AnalyzerCommand.kt#L73-L76). As you only do an `in` check, you wouldn't even need to call `.distinct()` on the list.

Reply:
Done.